decidim-templates/db/migrate/20221006184905_migrate_templateable.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MigrateTemplateable < ActiveRecord::Migration[6.0]
+  def self.up
+    Decidim::Templates::Template.find_each do |template|
```

Comment:
I think we should use one of the strategies from https://github.com/decidim/decidim/discussions/8068 so we don't have problems in the future. How do you see it?